### PR TITLE
Add CONTRIBUTING.md to repo

### LINKED ISSUE #26: Add CONTRIBUTING.md to repo

### PR DESCRIPTION
**Pull request by JuanPabloDiaz**
Add CONTRIBUTING.md with contribution guidelines

closes #26

I did a step-by-step instruction on how to contribute but if you prefer, I can do this instead:

### Set up

```shell
git clone
cd unitycatalog-ui
yarn install
yarn start
```

### CODE REVIEW

**Comment by JuanPabloDiaz** [comment]
Also, is the repo going to have a CODE_OF_CONDUCT.md so I can add it?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+4. Create your branch
+
+   ```sh
+   git checkout -b <your_branch_name>
+   ```
+
+5. Install the dependencies and start the development server
+
+   ```sh
+    yarn
```
Comment (robertgv) [inline CONTRIBUTING.md:42]: There is a space at the start of "yarn"

Comment (JuanPabloDiaz) [reply CONTRIBUTING.md:42]: Thank you for catching that.

**Review by Romanize — APPROVED** [review]

**Review by MrPowers — APPROVED** [review]
LGTM, thanks @JuanPabloDiaz!

**Comment by dennyglee** [comment]
@JuanPabloDiaz concerning the code_of_conduct, should we create a new `CODE_OF_CONDUCT.md` verbatim and/or point to the [unitycatalog/unitycatalog/CODE_OF_CONDUCT.md](https://github.com/unitycatalog/unitycatalog/blob/main/CODE_OF_CONDUCT.md)

**Comment by JuanPabloDiaz** [comment]
> @JuanPabloDiaz concerning the code_of_conduct, should we create a new `CODE_OF_CONDUCT.md` verbatim and/or point to the [unitycatalog/unitycatalog/CODE_OF_CONDUCT.md](https://github.com/unitycatalog/unitycatalog/blob/main/CODE_OF_CONDUCT.md)

I recommend pointing to the existing CODE_OF_CONDUCT.

**Comment by dennyglee** [comment]
Cool, @JuanPabloDiaz, up for creating the PR and I'll review it?

**Pull request merged**